docs/conf.py

```diff
@@ -433,10 +433,10 @@ def setup(app):
 #
 # texinfo_no_detailmenu = False
 
-mathjax_config = {
+mathjax3_config = {
     'tex2jax': {'inlineMath': [['$', '$'], ['\\(', '\\)']]},
```

Comment:
@kratsg Things are working, so I should probably leave it alone, but from poking around in the mathjax docs I don't understand how `tex2jax` works as I can't find it documented well. Thoughts?

Reply:
It's just a preprocessor: https://docs.mathjax.org/en/v2.7-latest/options/preprocessors/tex2jax.html

Reply:
Thanks!